example/lib/main.dart

```diff
@@ -15,6 +15,7 @@
 import 'package:flutter/foundation.dart'
     show debugDefaultTargetPlatformOverride;
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
```

Comment:
What prompted this change? It doesn't seem related to the PR.

Reply:
I'm not sure why this is marked as a change. This was already there and we need it to get the RawKeyboard*.